.env.test.example

```
@@ -34,11 +38,11 @@ PGVECTOR_HNSW_EF_SEARCH_DEFAULT=96
 MEMORIES_RETENTION_DAYS=180
 
 # ---- Upstash / QStash (HTTP-only) -----------------------------------------
-UPSTASH_REDIS_REST_URL=https://test-redis.upstash.io
-UPSTASH_REDIS_REST_TOKEN=test-upstash-token
-QSTASH_TOKEN=test-qstash-token
-QSTASH_CURRENT_SIGNING_KEY=test-qstash-current
-QSTASH_NEXT_SIGNING_KEY=test-qstash-next
+UPSTASH_REDIS_REST_URL=
+UPSTASH_REDIS_REST_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The UPSTASH_REDIS_REST_TOKEN key should go before the UPSTASH_REDIS_REST_URL key

(UnorderedKey)